elements/bulbs-pollzone-map/bulbs-pollzone-map.js

```diff
@@ -0,0 +1,121 @@
+import { 
+  BulbsHTMLElement, 
+  registerElement 
+} from 'bulbs-elements/register';
+
+import _ from 'lodash';
+
+import * as d3 from 'd3';
+import topojson from 'topojson';
+import Datamap from 'datamaps';
+
+import './bulbs-pollzone-map.scss';
+
+class BulbsPollzoneMap extends BulbsHTMLElement {
+  attachedCallback () {
+    let $this = $(this).find('.map');
+    let $data = $this.data('map-data');
+
+    this.legendContainer = $(this).find('.legend')[0];
+    this.statesData = $data.votes;
+    this.questionsData = $data.questions;
+    this.setFills();
+    this.initMap();
+  }
+
+  setFills () {
+    this.fills = {
+      defaultFill: '#006B3A',
+    };
+
+    // add colors to keyed fills list
+    _.forEach([
```

Comment:
it'll definitely happen at some point. I'd just rather have that be the task in itself, since there are (likely) other elements that will need the same refactor and it'd be easier to test if they're refactored at the same time